src/crypto/crypto_cipher.h

```diff
@@ -249,7 +249,7 @@ class CipherJob final : public CryptoJob<CipherTraits> {
       v8::Local<v8::Value>* result) override {
     Environment* env = AsyncWrap::env();
     CryptoErrorStore* errors = CryptoJob<CipherTraits>::errors();
-    if (out_.size() > 0) {
+    if (out_.size() > 0 || (!out_.size() && errors->Empty())) {
```

Comment:
```suggestion
    if (out_.size() > 0 || errors->Empty()) {
```

`size()` returns a `size_t` which is always `>=0` so the `!out_.size()` is redundant here.